src/extension/settingsHelper.ts

```diff
@@ -52,6 +52,18 @@ export class SettingsHelper {
         return workspaceFolder.uri.fsPath;
     }
 
+    /**
+     * Get the React Native Global Command Name
+     */
+    public static getReactNativeGlobalCommandName(uri: vscode.Uri): string {
+        const workspaceConfiguration = vscode.workspace.getConfiguration("react-native-tools", uri);
+        if (workspaceConfiguration.has("reactNativeGlobalCommandName")) {
+            return ConfigurationReader.readString(workspaceConfiguration.get("reactNativeGlobalCommandName"));
+        }
+
+        return "";
```

Comment:
why not return null / undefined ?  That's easier to check for validity than an empty string.

Reply:
Thanks for pointing, I'll take care of it.